vmdb/db/migrate/20150401090146_add_systemd_and_openstack_related_columns_to_system_services.rb

```diff
@@ -0,0 +1,9 @@
+class AddSystemdAndOpenstackRelatedColumnsToSystemServices < ActiveRecord::Migration
+  def change
+    add_column :system_services, :load, :string
+    add_column :system_services, :active, :string
+    add_column :system_services, :sub, :string
+    add_column :system_services, :openstack_host_service_id, :bigint
+    add_index  :system_services, :openstack_host_service_id
```

Comment:
When we added systemd support to VM fleecing, we added a serialized column to represent dependencies, which could be populated as needed depending on the type of system services in question. Instead of adding additional systemd-specific data columns to the table, maybe we should go with the same serialized col approach. This way subclasses of system_services can use the column as needed.

Adding a new column would be redundant, but leveraging the existing column would require changing the existing VM fleecing code.

@movitto what's your take on this?